health-services/project-factory/src/server/utils/onGoingCampaignUpdateUtils.ts

```
@@ -356,38 +354,47 @@ async function fetchProjectsWithProjectId(request: any, projectId: any, tenantId
-    const projectSearchResponse = await httpRequest(config?.host?.projectHost + config?.paths?.projectSearch, projectSearchBody, projectSearchParams);
-    if (projectSearchResponse?.Project && Array.isArray(projectSearchResponse?.Project) && projectSearchResponse?.Project?.length > 0) {
-      return projectSearchResponse;
-    }
-    else {
-      return null;
-    }
-  } catch (error: any) {
-    throwError("PROJECT", 500, "PROJECT_SEARCH_ERROR")
+    };
+
+    logger.info("Project search params: " + JSON.stringify(projectSearchParams));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)